phoenicis-javafx/src/main/resources/org/phoenicis/javafx/themes/default/main.less

```diff
@@ -354,42 +366,42 @@
 }
 
 .listChooser {
-    -fx-spacing: 0.2em;
```

Comment:
IntelliJ formatting shortcut. It would be nice if our maven formatter plugin could format css files too :/